projects/plugins/boost/app/assets/src/js/features/critical-css/critical-css-meta/critical-css-meta.tsx

```diff
@@ -45,7 +48,9 @@ export default function CriticalCssMeta() {
 				) }
 			/>
 
-			<RegenerateCriticalCssSuggestion regenerateReason={ regenerateReason } />
+			{ ! showFatalError && (
+				<RegenerateCriticalCssSuggestion regenerateReason={ regenerateReason } />
+			) }
```

Comment:
@haqadn do you think it makes more sense to have the `RegenerateCriticalCssSuggestion` module inside Status? We do have a check for `isCloud`. That way, we wouldn't have to tell status if it's a fatal error or not - it could figure that out on its own.

Reply:
No strong objections. My only concern is that the Status component is already quite complex and seems like it's doing a lot of things. Maybe we can do a refactor of the `Status` some other time. For now, if you feel like the `RegenerateCriticalCssSuggestions` belongs inside `Status`, go ahead.

Reply:
I felt the same way as you did about `Status` - it's bloated enough. The critical/cloud css meta component is going to just be a wrapper around `Status`. Let's leave it out for now.